CHANGELOG.md

```diff
@@ -9,5 +9,9 @@ All notable changes to this project are documented in this file.
 - The `cmake/AddUninstallTarget.cmake` file
 - The `cmake/FindEigen3.cmake` file
 - The `cmake/InstallBasicPackageFiles.cmake` file
+- Implement the first version of the `OptimalControlUtilities` library
+
+### Changed
+- `README.md` file describing how to build and install the library
```

Comment:
I think the "Changed" section of the changelog only captures the changes w.r.t. to the previous release, so if there are changes accross commits that will end up in the same release there is no need to document them.

Reply:
Thanks done